eng/targets/Helix.props

```diff
@@ -10,33 +10,20 @@
 
   <PropertyGroup>
     <HelixTimeout>00:30:00</HelixTimeout>
-    <!-- Do not attempt to override global property. -->
-    <RunQuarantinedTests Condition=" '$(RunQuarantinedTests)' == '' ">false</RunQuarantinedTests>
-
-    <IsArm64HelixQueue>false</IsArm64HelixQueue>
-    <IsWindowsHelixQueue>false</IsWindowsHelixQueue>
-    <IsMacHelixQueue>false</IsMacHelixQueue>
-
     <HelixTestName>$(MSBuildProjectName)--$(TargetFramework)</HelixTestName>
     <LoggingTestingDisableFileLogging Condition="'$(IsHelixJob)' == 'true'">false</LoggingTestingDisableFileLogging>
     <NodeVersion>16.11.0</NodeVersion>
+
     <!-- Have all tests depend on the latest runtimes until we get a net7.0 SDK -->
     <TestDependsOnAspNetPackages>true</TestDependsOnAspNetPackages>
     <TestDependsOnAspNetRuntime>true</TestDependsOnAspNetRuntime>
   </PropertyGroup>
 
-  <!-- Override a few properties in innermost (Publish) build for Helix submission. -->
-  <PropertyGroup Condition=" '$(HelixTargetQueue)' != '' ">
-    <IsArm64HelixQueue Condition="$(HelixTargetQueue.Contains('Arm64'))">true</IsArm64HelixQueue>
-    <IsWindowsHelixQueue Condition="$(HelixTargetQueue.Contains('Windows'))">true</IsWindowsHelixQueue>
-    <IsMacHelixQueue Condition="$(HelixTargetQueue.Contains('OSX'))">true</IsMacHelixQueue>
-  </PropertyGroup>
-
-  <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework'">
+  <ItemGroup Condition=" '$(TargetFrameworkIdentifier)' == '.NETFramework' OR '$(IsWindowsOnlyTest)' == 'true' ">
```

Comment:
I'm guessing there's no way to cleanly do an if/else type of thing, since I'm reading these two item groups as basically an if/else right?

Reply:
`<Choose/>` provides the main `if`/`else` option in `msbuild` but `<Choose/>` conditions get evaluated a bit too early to make their use simple for properties changed in TFM-specific project builds. I guess we could move this to Helix.targets but I'm less worried about that than the previous confusion between the two .props files.

Reply:
Made the change to Helix.targets and `<Choose/>` in second commit (along w/ fixing Helix.Common.props).